activerecord/lib/active_record/core.rb

```diff
@@ -47,8 +47,20 @@ def self.destroy_association_async_job
       class_attribute :destroy_association_async_batch_size, instance_writer: false, instance_predicate: false, default: nil
 
       ##
-      # Contains the database configuration - as is typically stored in config/database.yml -
-      # as an ActiveRecord::DatabaseConfigurations object.
+      # Sets the database configuration:
+      #
+      #   ActiveRecord::Base.configurations = {
+      #     "development" => { "adapter" => "abstract", "database" => "dev-db" }
+      #   }
+      def self.configurations=(config)
+        @@configurations = ActiveRecord::DatabaseConfigurations.new(config)
+      end
+      self.configurations = {}
+
+      ##
+      # Contains the database configuration - as is typically stored in
+      # config/database.yml - as an ActiveRecord::DatabaseConfigurations
+      # object.
```

Comment:
I wonder if you don't change the wrapping does this diff get cleaner.

Also, would it make sense to include a link/ref to AR::DatabaseConfigurations?

Reply:
Thanks for the review @zzak ! I’m mostly offline for a couple of weeks. I’ll have a look when I get back.